napari/utils/translations.py

```diff
@@ -210,11 +218,11 @@ def __new__(
 
     def __init__(
         self,
-        domain: Optional[str] = None,
+        domain: str,
         msgctxt: Optional[str] = None,
         msgid: Optional[str] = None,
         msgid_plural: Optional[str] = None,
-        n: Optional[str] = None,
+        n: Optional[int] = None,
```

Comment:
What is `n`? If you investigate this code then maybe you could add docstrings?

Reply:
I'm not sure exactly, but this typing fix is definitely correct. I'd advocate for docstrings being beyond scope of this PR.

Reply:
The docstring is out of scope. But I do not understand this code. some comments may help. Or this PR may need to wait longer for some core-dev who is familiar with translations.

Reply:
I added a signpost to https://docs.python.org/3/library/gettext.html in the docstring, which is where these arguments are explained.

Reply:
`n` is the number of items, some language have pluralizations that depends on the number of things.

https://lingohub.com/blog/2019/02/pluralization

Reply:
`n` is the number of items, some language have pluralizations that depends on the number of things.

https://lingohub.com/blog/2019/02/pluralization